### PR TITLE
Use link names in temperature messages

### LINKED ISSUE #315: Incorrect frame_id values for Temperature messages?
The current implementation (`kinetic-devel`) uses the joint names as values for the `header.frame_id` when publishing `Temperature` messages for the joint (motor) temperatures:

https://github.com/ros-industrial/ur_modern_driver/blob/6b818fa149bcd070131d6b04e5d97e2f67d28684/src/ros/rt_publisher.cpp#L94-L107

`joint_names_` is populated here:

https://github.com/ros-industrial/ur_modern_driver/blob/6b818fa149bcd070131d6b04e5d97e2f67d28684/include/ur_modern_driver/ros/rt_publisher.h#L72-L75

with the defaults coming from here:

https://github.com/ros-industrial/ur_modern_driver/blob/6b818fa149bcd070131d6b04e5d97e2f67d28684/include/ur_modern_driver/ros/rt_publisher.h#L37-L38

`frame_id`s can never be set to names of `joint`s, only to `link` names, as only the latter will correspond to TF frames.

The intent of the publisher is of course to publish joint temperatures, so using `joint_names_` seems like the logical thing to do, but `header.frame_id` just cannot be used that way, as it violates convention and thus potentially breaks applications.

### PR DESCRIPTION
**Pull request by fmauch**
This resolves #315

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
See mine: https://github.com/ros-industrial/ur_modern_driver/issues/315#issuecomment-507614775

;)

File: include/ur_modern_driver/ros/rt_publisher.h [hunk 1]
```diff
@@ -36,6 +36,9 @@ using namespace tf;
 
 const std::string JOINTS[] = { "shoulder_pan_joint", "shoulder_lift_joint", "elbow_joint",
                                "wrist_1_joint",      "wrist_2_joint",       "wrist_3_joint" };
+// links with origin in the joints configured above.
+const std::string LINKS[] = { "shoulder_link", "upper_arm_link", "forearm_link",
+                              "wrist_1_link",  "wrist_2_link",   "wrist_3_link" };
```
[the remark this answers is not in the document]
Comment (fmauch) [reply include/ur_modern_driver/ros/rt_publisher.h]: That was my thinking, as well.

**Comment by gavanderhoorn** [comment]
Just waiting for Travis now. Has been very slow today ..

**Comment by gavanderhoorn** [comment]
Thanks @fmauch :+1:

**Pull request merged**